### PR TITLE
[JUJU-1471] Switch from python-based to bash-based version of nw-constraints-aws

### PR DESCRIPTION
**Pull request by anvial**
This is the first time I have generated new bash-based jobs. So need to check carefully.

### CODE REVIEW

File: tools/gen-wire-tests/juju.config [hunk 1]
```diff
@@ -19,6 +19,7 @@ folders:
     - ck
   skip-lxd:
     - caasadmission
+    - constraints
```
Comment (wallyworld) [inline tools/gen-wire-tests/juju.config:22]: we do want to run this on lxd as well so remove this line and regenerate

File: tools/gen-wire-tests/juju.config [hunk 2]
```diff
@@ -78,6 +81,7 @@ folders:
     - branches
     - charmhub
     - cli
+    - constraints
```
Comment (wallyworld) [inline tools/gen-wire-tests/juju.config:84]: i think having a constraints test on microk8s is something we'd want

**Review by wallyworld — APPROVED** [review]
If you include the lxd and microk8s tests, we probably don't need the change to the yaml simplify lint config

File: tools/gen-wire-tests/juju.config (hunk 1 above)
Comment (anvial) [reply tools/gen-wire-tests/juju.config:22]: for now, we have constraints test only for aws (as was in old python-based).

If we wish, we need firstly add new tests in the constraints suite (now it has only "run_contstraints_aws")

File: tools/gen-wire-tests/juju.config (hunk 2 above)
Comment (anvial) [reply tools/gen-wire-tests/juju.config:84]: I need to check what constraints we are supporting for microk8s... and, firstly add them to the constraints test suite

Comment (anvial) [reply tools/gen-wire-tests/juju.config:81]: Constrains tests for microk8s are not implemented yet. So we should leave this 'skip'

**Pull request merged**